lightkurve/periodogram.py

```diff
@@ -749,14 +749,8 @@ def from_lightcurve(lc, minimum_frequency=None, maximum_frequency=None,
                              nterms=nterms, **kwargs)
             power = LS.power(frequency, method=method, normalization='psd')
 
-        # Normalise the according to Parseval's theorem
-        norm = np.std(lc.flux * 1e6)**2 / np.sum(power)
-        power *= norm
-
-        power = power * (cds.ppm**2)
-
         # Rescale power to units of ppm^2 / [frequency unit]
-        power = power / fs
+        power *=  2./(len(time)*oversample_factor*fs) * (cds.ppm**2)
```

Comment:
Could you add a comment above this line explaining how the normalisation is done (just a single line, as before)? Also possibly include a 'why', which I neglected to do in the original code.

Reply:
Added something slightly more descriptive

Reply:
Awesome, thanks Keaton!